app/src/debug/java/net/squanchy/support/debug/DebugActivity.kt

```diff
@@ -41,6 +47,9 @@ class DebugActivity : AppCompatActivity() {
         buttonResetOnboarding.setOnClickListener { resetOnboarding() }
 
         notificationCreator = NotificationCreator(this)
+
+        findViewById<Button>(R.id.freezeTime).setOnClickListener { freezeTime() }
```

Comment:
Would require it of boyscouting, but we could use kotlinx synthetics instead of `findViewById`

Reply:
I tried, it doesn't work on other source sets than main (there's also a bug open somewhere which has been apparently fixed but not released)

Reply:
TIL! Ok :)